tests/msg_avail/main.c

```diff
@@ -34,28 +36,27 @@ int main(void)
 
     msg_init_queue(msg_queue, MSG_QUEUE_LENGTH);
 
+    puts("[START]");
```

Comment:
and more over, most output is only useful when running tests manually, but not if we (hopefully soon) integrate them into an automatic testing suite with the CI.